docs/configuration-reference.md

```diff
@@ -153,6 +153,13 @@ define one instance.
 # Configure the directory used by instances to store their WAL.
 [wal_directory: <string> | default = ""]
 
+# Configures how long ago an abandoned (not associated with an instance) WAL
+# may be written to before being eligible to be deleted
+[wal_cleanup_age: <duration> | default = "12h"]
+
+# Configures how often checks for abandoned WALs to be delete are performed
+[wal_cleanup_period: <duration> | default = "30m"]
```

Comment:
Sorry for adding this feedback on the docs, but can we make the WAL cleanup optional? If the cleanup period is configured to `0s`, let's disable it.

Reply:
Sure. I was thinking the same thing, that this should probably have an off switch